roles/pmm/tasks/main.yaml

```diff
@@ -16,21 +16,16 @@
           ansible.builtin.file:
             path: '{{ pmm_container_env_file | dirname }}'
             state: directory
-            owner: root
-            group: root
+            owner: '{{ ansible_env.USER }}'
             mode: u=rwX,g=rX,o=X
 
         - name: Manage the container environment file
           ansible.builtin.template:
             src: env-pmm-server.j2
             dest: '{{ pmm_container_env_file }}'
-            owner: root
-            group: root
+            owner: '{{ ansible_env.USER }}'
             mode: u=rw,a=r
```

Comment:
The `mode` needs to be changed as you have removed `sudo` and so permissions will be checked